### PR TITLE
Bug 393 authorization selinux

### PR DESCRIPTION
**Pull request by atye**
# Description

Updates the prerequisites for the Authorization RPM deployment to install container-selinux and a specific version of k3s-selinux.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/393 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/authorization/deployment/rpm/_index.md [hunk 1]
```diff
@@ -19,7 +19,21 @@ The CSM for Authorization proxy server requires a Linux host with the following
 
 These packages need to be installed on the Linux host:
 - container-selinux
-- https://rpm.rancher.io/k3s/stable/common/centos/7/noarch/k3s-selinux-0.4-1.el7.noarch.rpm
+- k3s-selinux-0.4-1
+
+Use the appropriate package manager on the machine to install the packages.
```
Comment (sharmilarama) [inline content/docs/authorization/deployment/rpm/_index.md:24]: Can we provide specific instructions for installing on dark sites? yum will not work in that case.

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. No edits are required.

File: content/docs/authorization/deployment/rpm/_index.md (hunk 1 above)
Comment (atye) [reply content/docs/authorization/deployment/rpm/_index.md:24]: Updated. We decided to leave it fairly general.

**Review by shaynafinocchiaro — APPROVED** [review]

File: content/docs/authorization/deployment/rpm/_index.md [hunk 2]
```diff
@@ -35,6 +35,12 @@ yum install -y container-selinux
 
 yum install -y https://rpm.rancher.io/k3s/stable/common/centos/8/noarch/k3s-selinux-0.4-1.el8.noarch.rpm
 
+### Dark Sites
+
+For environments where `yum` will not work, obtain the supported version of container-selinux for your OS version and install it.
```
Comment (sharmilarama) [inline content/docs/authorization/deployment/rpm/_index.md:40]: Is there a landing page link we can add here for the download?

Comment (atye) [reply content/docs/authorization/deployment/rpm/_index.md:40]: Sort of. There is https://github.com/containers/container-selinux/releases and https://pkgs.org/download/container-selinux. The GitHub releases page isn't clear about supported versions but the second link is. Should we use the second link as an example landing page?

Comment (atye) [reply content/docs/authorization/deployment/rpm/_index.md:40]: Updated to use https://centos.pkgs.org as an example to download from.

**Review by sharmilarama — APPROVED** [review]

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**